tools/iam_migrations/ldap/Cargo.toml

```diff
@@ -40,3 +40,6 @@ kanidm_utils_users = { workspace = true }
 [build-dependencies]
 clap = { workspace = true, features = ["derive"] }
 clap_complete = { workspace = true }
+
+[dev-dependencies]
+tracing-forest = "0.1.6"
```

Comment:
Shouldn't it be sketching?

Reply:
yep, changed. (to the sketching version which makes the exact same call 🤣 )